add expand extents (#171)
GeoscienceAustralia/dea-knowledge-hub: docs/guides/reference/ard-processing-extent.md

# DEA ARD processing extent

> ## Background

Reviewer: @OHehir-Aaron Can I delete this 'Background' heading so that effectively this section becomes the 'introduction' section.

Author: Yep!

> We are extending our baseline satellite data to include Christmas Island, Cocos (Keeling) Islands, Coral Sea Islands, Heard and McDonald Islands, Norfolk Island and Islands in the Torres Strait. Additionally, we are processing imagery over shallow Ocean to support bathymetry and oceanography science.

Reviewer 2: ```suggestion
We are extending our baseline satellite data coverage to include Christmas Island, Cocos (Keeling) Islands, Coral Sea Islands, Heard and McDonald Islands, Norfolk Island and Torres Strait islands. Additionally, we are processing imagery over shallow ocean areas to support bathymetry and oceanographic science.

```

> The new data was requested by and will support Indigenous Communities in the Torres Straits in addition to Government agencies who seek to report on Ocean Ecosystem Accounts and manage the full extent of our Marine Parks. 

Reviewer 2: ```suggestion
The new data was requested by and will support Indigenous communities in the Torres Strait, in addition to Government agencies who seek to report on Ocean Ecosystem Accounts and manage the full extent of our Marine Parks.

```

> The update is in support of DEA's expansion of the Summary Product Grid to shift the origin point to the south-west of the current position. The expanded Area of Interest will support several DEA product releases and version updates throughout 2024.

Reviewer 2: ```suggestion
The update will support the expansion of [DEA's Summary Product Grid](https://knowledge.dea.ga.gov.au/guides/reference/collection_3_summary_grid/), including a shift in the origin point to the south-west of its current position. The expanded Area of Interest will support several DEA product releases and version updates throughout 2024.
```

The update is in support of DEA's expansion of the Summary Product Grid to shift the origin point to the south-west of the current position. The expanded Area of Interest will support several DEA product releases and version updates throughout 2024.

> ## Expanded locations

Reviewer: @OHehir-Aaron Could this section start with some brief text that describes what this section means.

Author: It currently has 'background' is that sufficient?

> Here is the new full list of locations that are supported.

Reviewer: @OHehir-Aaron I reworded this from 'Expanded locations' to be clear that this was a full list of locations rather than only the new ones.

:::{list-table}
:header-rows: 1

* - Locality

>   - Tiles

Reviewer: @OHehir-Aaron I added column headers to this table but I'm not sure I have got the terminology right. 'Locality' and 'Tiles' - please change these if they are incorrect.

Author: The terminology for Landsat is 'scenes' and for Sentinel 2 it is 'tiles'. Scenes are Path/Row and tiles are MGRS grid.

  - 099-066 
* - Daru, Western Province Papua New Guinea and Northern Warrior Reefs
  - 098-066, 54LZR
* - Elizabeth Reef
  - 085-081, 57JVG, 57JWG 
* - Middleton Reef
  - 085-080, 57JVH, 57JWH 
* - Frederick Reef, Queensland
  - 088-075, 56KQB, 57KTS, 56KQA 
* - Coral Sea – Scenes with Ocean depth less than 200m
  - 089-072, 088-074, 088-076, 54LZL, 55KEA, 55KFV, 55KGA, 55KGV, 55KHA, 55KHT, 55KHU, 55LBF, 55LBJ, 55LBK, 55LCJ, 55LDC, 55LDE, 56KKC, 55KKD, 56KKF, 56KLC, 56KMV, 56KNU, 56KQA, 56KQB, 56KQF, 56KQG, 56KQV, 56KRB, 56KRF, 56KRG, 57KTA, 57KTB, 57KTS, 57KTS 
* - Gulf of Carpentaria
  - 101-069, 53LRC, 54LTH 
* - Arafura Sea
  - 102-067, 104-067, 106-067, 107-068, 107-069, 52LEM, 52LEN, 52LFP, 52LHP, 53LLJ, 53LPJ, 53LQH, 53LQJ 
* - Timor Sea
  - 51LVD, 51LWD, 51LWF, 51LXE, 51LXF, 51LYF, 51LZF, 52LBL, 52LCL, 52LDK 
* - Indian Ocean
  - 116-075, 49HGB, 49HGC, 49JFL, 49KGS, 49KHT, 50HKG, 50HKH, 50HKJ, 50HMF, 50HNF, 50JKL, 50KKC 
* - Great Australian Bight
  - 52HBI, 52HEK, 52HFK, 53HKD, 53HLC, 53HNA 
* - Augusta, Western Australia
  - 113-084 
* - Tasman Sea
  - 089-088, 089-091, 090-091, 55GFM 
* - Bass Strait
  - 54HXB 

:::

## Quality and metadata 

> The quality of ancillary inputs used in ARD generation in offshore locations is limited in comparison to the higher quality ancillary inputs used on continental Australia. Quality information for ancillary inputs is found in the proc-info (processing information) YAML metadata file within the dataset. The tier list is USER, FALLBACK, and DEFINITIVE, with the latter being the highest level of quality. Unfortunately, this information is not available to users via the datacube, so we have added an additional metadata tag to delineate data delivered in this update: `final_ancillaries = "nonstandard"`

Reviewer 2: ```suggestion
The quality of ancillary inputs used in ARD generation in offshore locations is limited in comparison to the higher quality ancillary inputs used on continental Australia. Quality information for ancillary inputs is found in the `proc-info` (processing information) YAML metadata file within the dataset. The tier list is `USER`, `FALLBACK`, and `DEFINITIVE`, with the latter being the highest level of quality. Unfortunately, this information is not available to users via the datacube, so we have added an additional metadata tag to delineate data delivered in this update: `final_ancillaries = "nonstandard"`

```

```python
dc.find_datasets(product="ga_ls8c_ard_3", limit=10, final_ancillaries="nonstandard")
```

## Geometric quality assessment issues 

> [ARD](/guides/about/glossary/#ard) is a Surface Reflectance product which is derived from USGS or ESA level 1 products. The level 1 product is geometrically corrected such that the product is 'ortho-rectified' which results in enabling acquisitions from different dates to be spatially overlaid for assessment through time. The ARD product provides a comprehensive quality assurance of geometric correction known as GQA, which is found in the dataset and processing information metadata.

Reviewer 2: ```suggestion
[ARD](/guides/about/glossary/#ard) is a Surface Reflectance product which is derived from USGS or ESA Level 1 products. The Level 1 product is geometrically corrected such that the product is 'ortho-rectified' which results in enabling acquisitions from different dates to be spatially overlaid for assessment through time. The ARD product provides a comprehensive quality assurance of geometric correction known as GQA, which is found in the dataset and processing information metadata.

```

> This update features the addition of Landsat scenes and Sentinel 2 tiles over islands and reefs. Some of these Islands, such as Christmas Island or Islands South of Papua New Guinea, are at extreme high or low latitudes and are affected by persistent cloud. This presents challenges when correlating the reference image against source images, to provide statistical geometric comparison results.

Reviewer 2: ```suggestion
This update features the addition of Landsat scenes and Sentinel 2 tiles over islands and reefs. Some of these islands, such as Christmas Island or islands south of Papua New Guinea, are at extreme high or low latitudes and are affected by persistent cloud. This presents challenges when correlating the reference image against source images to provide statistical geometric comparisons.

```

## BRDF quality issues 

[BRDF](/guides/about/glossary/#brdf) is a solar illumination correction which adjusts reflectance intensity in differing amounts in differing directions. The light that comes from the Sun, reflects off the Earth and is received by the sensor is adjusted based on viewing/solar angle geometries of MODIS data.

> Sometimes over offshore locations we have found BRDF values to be unphysical. We have implemented a BRDF solution over the offshore locations, which screens out unphysical BRDF values and excludes them from the solar illumination correction.

Reviewer 2: "Unphysical" might be confusing to our users. Can we explain this in more detail?

Author: Hi @robbibt , I have updated this section after consulting with Fuqin and Aman.

## Water vapor 

No ancillary source dataset difference between mainland and offshore locations.

## Aerosol 

> A constant value of 0.06 Aerosol Optical Depth (AOD) is used as ancillary input as we do not have aerosol data for these extended locations. The constant value is based on the assumption that the average AOD data in Australian continental is around the 0.05, but islands are surrounded by the sea and impacted by the particle of salt water, so setting AOD as 0.06 is reasonable.

Reviewer 2: "continental" what? Might be a missing word here

Please check my correction for "salt water particle" to make sure this makes sense

```suggestion
A constant value of 0.06 Aerosol Optical Depth (AOD) is used as ancillary input as we do not have aerosol data for these extended locations. This constant value is based on the assumption that the average AOD data in Australian continental is around the 0.05, but islands are surrounded by the sea and impacted by salt water particles, so setting AOD as 0.06 is reasonable.

```
